joints: ['head_pan_joint']

head_pan_joint:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: head_pan_joint
    joint_speed: 2.0
    motor:
        id: 16
        init: 512
        min: 0
        max: 1023
